templates/careers/application/_withdrawal_notification-email.html

```diff
@@ -2,7 +2,7 @@
   <p>Dear {{ hiring_lead_name }},</p>
   <p>This is an automated email to inform you that {{ applicant_name }} has withdrawn their application for the {{ position }} position.</p>
   <p>The candidate was in {{ current_stage["name"] }} stage.</p>
-  <p>If this is an interview stage, please make sure you check the activity feed of the candidate in Greenhouse and delete any pending interviews directly from Google Calendar.</p>
+  <!-- <p>If this is an interview stage, please make sure you check the activity feed of the candidate in Greenhouse and delete any pending interviews directly from Google Calendar.</p> -->
```

Comment:
If we no longer need this, would be best to remove it completely. We have Git history always to take back content that we've removed